File: Python/Disk.py
class Disk:
    __type = ""     # menampung tipe
    __capacity = "" # menampung capacity
    __price = 0     # menampung harga

    # constructor
    def __init__(self, type = "", capacity = "", price = 0):
        self.__type = type
        self.__capacity = capacity
        self.__price = price

    # setter and getter
    def setType(self, type):
        self.__type = type
    
    def getType(self):
        return self.__type

    def setCapacity(self, capacity):
        self.__capacity = capacity

    def getCapacity(self):
        return self.__capacity
    
    def setPrice(self, price):
        self.__price = price
    
    def getPrice(self):
        return self.__price

    # method
    def printDisk(self):
        print("Disk Type        : " + str(self.getType()))
        print("Disk Capacity    : " + str(self.getCapacity()))
        print("Disk Price       : Rp " + str(self.getPrice()) + ",-")

File: PHP/Disk.php
<?php

class Disk {
    private $type;      // menampung tipe
    private $capacity;  // menampung kapasitas
    private $price;     // menampung harga

    // constructor
    function __construct($type, $capacity, $price) {
        $this->type = $type;
        $this->capacity = $capacity;
        $this->price = $price;
    }

    // setter and getter
    function setType($type) {
        $this->type = $type;
    }

    function getType() {
        return $this->type;
    }

    function setCapacity($capacity) {
        $this->capacity = $capacity;
    }

    function getCapacity() {
        return $this->capacity;
    }

    function setPrice($price) {
        $this->price = $price;
    }

    function getPrice() {
        return $this->price;
    }

    // method
    function printDisk() {
        echo "Disk Type         : ". $this->getType(). "<br/>";
        echo "Disk Capacity     : ". $this->getCapacity(). "<br/>";
        echo "Disk Price        : Rp ". $this->getPrice(). ",-<br/>";
    }
}

?>